src/Behat/Gherkin/Node/ExampleNode.php

```diff
@@ -45,25 +45,31 @@ class ExampleNode implements ScenarioInterface
      * @var string
      */
     private $outlineTitle;
+    /**
+     * @var null|int
+     */
+    private $index;
 
     /**
      * Initializes outline.
      *
-     * @param string      $title
+     * @param string      $text         The entire row as a string, e.g. "| 1 | 2 | 3 |"
      * @param string[]    $tags
      * @param StepNode[]  $outlineSteps
      * @param string[]    $tokens
-     * @param integer     $line
-     * @param string|null $outlineTitle
+     * @param integer     $line         Line number within the feature file.
+     * @param string|null $outlineTitle Original title of the scenario outline.
+     * @param null|int    $index        The 0-based index of the row/example within the scenario outline.
```

Comment:
It says that the index is 0-based but it looks to be 1-based

Reply:
Ah, you're totally right.. the argument passed is `row + 1`..